ДЗ 1 и ДЗ 2

BoikoMaximAlexeevych opened the pull request:
Сделал оба задания в срок, но хотелось уже оба загрузить сюда

BoikoMaximAlexeevych committed:
ДЗ 1 и ДЗ 2

k-kondyukov commented on J3100 Бойко М/task1.cpp:
```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+using namespace std;
+int main(){
+char character = 257;
```
Поехали отступы. Нужно вот так:

```
int main(){
    char character = 257;
    std::cout << character + 1;

    return 0;
}
```

k-kondyukov commented on J3100 Бойко М/task2.cpp:
```diff
@@ -0,0 +1,39 @@
+std::cin >> number;
+switch (number)
+{
+case 1:
```
Тут такая же история с отступами

k-kondyukov commented:
Исправьте отступы, пожалуйста ^_^

BoikoMaximAlexeevych committed:
Добавил отступы.

Действительно добавил. И еще ноль в одно из заданий вписал.

k-kondyukov commented on J3100 Бойко М/task2.cpp:
```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+using namespace std;
+int main(){
+int number;
```
Теперь в этом скоупе не хватает отступа

BoikoMaximAlexeevych committed:
Добавил еще отступов

Теперь я как будто на питоне это написал, ей-богу))

k-kondyukov commented on J3100 Бойко М/task1.cpp:
```diff
@@ -0,0 +1,9 @@
+// Задание 1 (?)
+#include <iostream>
+using namespace std;
```
Не используем using namespace std

k-kondyukov commented on J3100 Бойко М/task2.cpp:
```diff
@@ -0,0 +1,41 @@
+		default:
+		std::cout << "Not a single-digit number";
+	}
+return 0;
```
Tab

BoikoMaximAlexeevych committed:
Больше отступов богу отступов

И меньше using namespace std.

BoikoMaximAlexeevych committed:
Мини-домашка 1

Определяем количество строк, начинающихся с определенного символа, с помощью лямбда-выражения.
